modules/custom/openy_upgrade_tool/src/OpenyUpgradeLogListBuilder.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Drupal\openy_upgrade_tool;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityListBuilder;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+/**
+ * Defines a class to build a listing of Openy upgrade log entities.
+ *
+ * @ingroup openy_upgrade_tool
+ */
+class OpenyUpgradeLogListBuilder extends EntityListBuilder {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildHeader() {
+    $header['name'] = $this->t('Config Name');
+    $header['updated'] = $this->t('Updated');
+    $header['user'] = $this->t('By User');
+    $header['status'] = $this->t('Status');
+    return $header + parent::buildHeader();
```

Comment:
let's use array_merge for edge cases , please

Reply:
Fixed